source/ShipInfoPanel.cpp

```diff
@@ -390,10 +391,18 @@ void ShipInfoPanel::DrawOutfits(const Rectangle &bounds, Rectangle &cargoBounds)
 	for(const auto &cat : GameData::GetCategory(CategoryType::OUTFIT))
 	{
 		const string &category = cat.Name();
+		if(category.empty())
+			continue;
 		auto it = outfits.find(category);
 		if(it == outfits.end())
 			continue;
 
+		auto validOutfits = std::ranges::filter_view(it->second,
+				[](const auto &o){return o->IsDefined() && !o->DisplayName().empty();});
```

Comment:
How long is the type of `validOutfits` going to be?
Isn't the type of `o` `const Outfit *`? `auto` doesn't seem necessary there, and the use of `&` is somewhat confusing because it looks like a reference to an object, but it's actually a reference to a pointer, so it still requires the "member of" operator. Unless the predicate for `filter_view` is required to take an argument of this type?
I think we usually only do one more indent for continuations like this?
```suggestion
		auto validOutfits = std::ranges::filter_view(it->second,
			[](const Outfit *o){return o->IsDefined() && !o->DisplayName().empty();});
```